pkg/component/watch.go

```diff
@@ -134,27 +134,29 @@ func WatchAndPush(client *occlient.Client, componentName string, applicationName
 		for {
 			select {
 			case event := <-watcher.Events:
+				isIgnoreEvent := false
 				changeLock.Lock()
 				glog.V(4).Infof("filesystem watch event: %s", event)
 
 				if !(event.Op&fsnotify.Remove == fsnotify.Remove || event.Op&fsnotify.Rename == fsnotify.Rename) {
 					stat, err := os.Lstat(event.Name)
 					if err != nil {
 						glog.Errorf("Failed getting details of the changed file %s", event.Name)
-						watchError = errors.Wrap(err, "unable to watch changes")
+						//watchError = errors.Wrap(err, "unable to watch changes")
```

Comment:
why is this error ignored? If there is a reason for it it should be explained in the comment

Reply:
The error is ignored because, in case of temporary files created by editors like vi or gedit that get deleted almost immediately and we get file not exists error sometimes. But if we feed this to error then the watch breaks, so I feel its better only to log it and leave it WDYT? @kadel